contracts/VAIController.sol

```diff
@@ -402,6 +416,17 @@ contract VAIController is VAIControllerStorageG2, VAIControllerErrorReporter, Ex
                 return (uint(Error.MATH_ERROR), 0);
             }
 
+            (, uint collateralFactorMantissa,) = Comptroller(address(comptroller)).markets(address(enteredMarkets[i]));
+            (vars.mErr, vars.sumSupply) = mulUInt(vars.sumSupply, collateralFactorMantissa);
+            if (vars.mErr != MathError.NO_ERROR) {
+                return (uint(Error.MATH_ERROR), 0);
```

Comment:
Instead of returning new error codes should we instead add requires to slowly migrate us off of returning errors?

Reply:
Agree, we should try to move off from "silent" errors. They make writing integration tests slower than in we just revert with error

Reply:
As discussed, we will resolve these in a separate PR.